docs/about/release-notes.md

```diff
@@ -21,15 +21,38 @@ The current and past members of the MkDocs team.
 * [@d0ugal](https://github.com/d0ugal/)
 * [@waylan](https://github.com/waylan/)
```

Comment:
@oprypin you might want to add us there?

Reply:
I noticed this but intentionally decided not to add "us". Let's see how this plays out longer term first.

Reply:
So, things played out quite calmly :)

[Add latest members of MkDocs maintenance team](https://github.com/mkdocs/mkdocs/pull/2661)